packages/oid4vci-holder/src/agent/OID4VCIHolder.ts

```diff
@@ -277,6 +278,7 @@ export class OID4VCIHolder implements IAgentPlugin {
         filter: [
           {
             identities: {
+              origin: IdentityOrigin.EXTERNAL,
```

Comment:
just making a comment for the default structure

Reply:
Commented it out with TODO pending feedback